routers/api/v1/convert/convert.go

```diff
@@ -73,6 +73,53 @@ func ToPublicKey(apiLink string, key *models.PublicKey) *api.PublicKey {
 	}
 }
 
+// ToGPGKey converts models.GPGKey to api.GPGKey
+func ToGPGKey(key *models.GPGKey) *api.GPGKey {
+	subkeys := make([]*api.GPGKey, len(key.SubsKey))
+	for id, k := range key.SubsKey {
+		subkeys[id] = &api.GPGKey{
+			ID:           k.ID,
+			PrimaryKeyID: k.PrimaryKeyID,
+			KeyID:        k.KeyID,
+			PublicKey:    k.Content,
+			Created:      k.Created,
+			Expires:      k.Expired,
+			//Emails:            emails,
+			//SubsKey:           subkeys,
```

Comment:
Those are empy in subkeys so they can be remove. I was keeping them commented to show they are not set explicitly but not really usefull.